src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import './App.css';
+import Menu from './components/MenuComponent';
+import Header from './components/Header';
+import { DISHES } from './shared/dishes';
+class App extends Component {
+  constructor(props)
+  {
+    super(props);
+    this.state={
+      dishes: DISHES,
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        {/*<div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <p>
+            Edit <code>src/App.js</code> and save to reload.
+          </p>
+          <a
+            className="App-link"
+            href="https://reactjs.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Learn React
+          </a>
+        </header>
+        
+    </div>*/}
+    <Header />
```

Comment:
nit: H in Header in lowercase

Reply:
"Header" is a component created by me, so should I change it to header but then it might conflict with header of react.

Reply:
no need it's okay I thought it's an HTML tag.